event-handler/remove-listener.js

```diff
@@ -14,7 +14,7 @@ function receiverListener (state, webhookNameOrNames, handler) {
 
   // remove last hook that has been added, that way
   // it behaves the same as removeListener
-  for (let i = state.hooks[webhookNameOrNames].length; i > 0; i--) {
+  for (let i = state.hooks[webhookNameOrNames].length - 1; i >= 0; i--) {
```

Comment:
Wow, good catch, thanks so much for investigating and sending a pull request, with tests 😻